web/src/main/resources/org/akaza/openclinica/i18n/notes.properties

```diff
@@ -18,7 +18,7 @@ X_event_wiht_definition = The study event with definition '
 a_new_user_account_has_been_created_for_you = A new user account has been created for you on the OpenClinica system.  Your login information follows:\n\n
 
 account_locked = Your User Name and Password combination has been Locked. Please contact your OpenClinica Administrator to have them unlock your account.
-
+session_locked = Unable logging in. Another user is already logged in on this web browser. Please have them log out before attempting to log in.
```

Comment:
please update the error message according to the updated AC

Reply:
done